src/components/02-organisms/Attachments/FileDownloader.js

```diff
@@ -0,0 +1,53 @@
+import React, { useContext, useMemo } from "react";
+import { injectIntl } from "react-intl";
+import PropTypes from "prop-types";
+import FetchHandler from "../../../FetchHandler";
+import { BackendContext } from "../../../context/backendContext";
+// import { BackendContext } from "./context/backendContext";
+
+const FileDownloader = props => {
+  const { state: fromBackend = {}, dispatch } = useContext(BackendContext);
+
+  const fetchSetup = useMemo(() => {
+    console.log(props.uuid);
+    if (props.uuid)
+      return [
+        {
+          key: "liitteet",
+          dispatchFn: dispatch,
+          urlEnding: props.uuid
+        }
+      ];
+    else return [];
+  }, [dispatch, props.file]);
+
+  const OpenFile = () => {
+    console.log(fromBackend);
+    // const blob = reader.result;
+    // let url = blob;
+    // let a = document.createElement("a");
+    // a.href = url;
+    // a.download = response.nimi + "." + response.tyyppi;
+    // a.click();
+    return <React.Fragment />;
+  };
+
+  return (
+    <React.Fragment>
+      <FetchHandler
+        fetchSetup={fetchSetup}
+        ready={
+          <React.Fragment>
+            <OpenFile />
+          </React.Fragment>
```

Comment:
`<React.Fragment>` can be removed.

Reply:
Nope, makes "nothing rendered" error.

Reply:
It's because of the OpenFile is a function and it's not called at that point. So let's keep the `<React.Fragment>´ there.